docs/amm.md

```diff
@@ -7,37 +7,44 @@ CoW AMM is an automated market maker running on top of CoW Protocol.
 > The code is not yet production ready and should not be used to handle large amounts of funds.
 > For technical aspects of the smart contract, reach out to us on [discord](https://discord.com/invite/cowprotocol) as we push towards production, or simply star the repository to be informed of progress!
 
-## How it works
+## Overview
 
 The AMM contract itself is a dedicated Safe multisig.
 It stores reserves of two tokens and allows anyone to create orders between these two tokens on CoW Protocol as long as the trade doesn't decrease the product of the reserves stored on the contract.
 
 The order is based on the `ComposableCoW` framework: this repository only specifies the code needed for the dedicated handler and the related price oracles.
 All the code for the `ComposableCoW` framework is [here](https://github.com/cowprotocol/composable-cow).
 
-The AMM also provides a view function (`getTradeableOrder`): this function doesn't impact which order is valid or not, but returns an order that can be traded in a way that tries to rebalance the AMM to align with the reference price of a price oracle.
+The AMM creates CoW Protocol order at regular time intervals.
+These orders are created with a dedicated contract function `getTradeableOrder`: it returns an order that can be traded in a way that tries to rebalance the AMM to align with the reference price of a price oracle.
+These orders are provided for convenience to CoW Swap solvers so that a basic CoW AMM usage does not need any dedicated implementation to tap into its liquidity.
```

Comment:
```suggestion
These orders are provided for convenience to CoW Swap solvers so that basic CoW AMM usage does not need any dedicated implementation to tap into its liquidity.
```